### PR TITLE
refine apy calculations

### PR DESCRIPTION
**Pull request by Opyet**

### CODE REVIEW

**Comment by vrtnd** [comment]
Provider error while running tests

File: src/adaptors/brinc-finance/index.js [hunk 1]
```diff
@@ -27,19 +27,20 @@ const brcContract = new web3.eth.Contract(BrcABI, BRC);
 const BLOCKS_PER_MONTH = 199384;
 const TOTAL_POOLS_WEIGHT = 568;
 const PoolNames = [
-  'BRC 7Days',
-  'BRC 30Days',
-  'BRC 90Days',
-  'BRC-gBRC 7Days',
-  'BRC-gBRC 30Days',
-  'BRC-gBRC 90Days',
+  'BRC (7Days)',
```
Comment (slasher125) [inline src/adaptors/brinc-finance/index.js]: please revert the change re (). 

on the frontend it will look as intended:
Screenshot 2022-08-31 at 23 28 56

**Comment by slasher125** [comment]
> Provider error while running tests

adaptor runs fine locally and on aws, bit sus that it breaks here

File: src/adaptors/brinc-finance/index.js (hunk 1 above)
Comment (Opyet) [reply src/adaptors/brinc-finance/index.js]: It's not really clear what your request is.

Do you want us to remove the ()? You asked to have them added in #48

Comment (slasher125) [reply src/adaptors/brinc-finance/index.js]: hey, sry: we made a few changes in the meantime, and we moved that info to the `poolMeta` field. i confirm: please remove () and keep it like it master

Comment (Opyet) [reply src/adaptors/brinc-finance/index.js]: They're removed now

**Comment by slasher125** [comment]
tested locally, runs fine. merging despite tests failing
